README.md

```diff
@@ -118,7 +118,7 @@ TODO: record and post the first lab walking through creation, execution and opti
     If you check that box, your action will now be publicly advertised to all of GitHub!
 14) **PSA:** The rest of this is optional. If you don't care about your action going fast, stop right here.
 15) Now we are going to make this **FAST** by pre-baking our container image. Go back to `templated-action` and edit
-    `./github/workflows/release-image.yml`. Change `docker.pkg.github.com/owner/` to use your repo owner for `owner`.
+    `./github/workflows/image-release.yml`. Change `docker.pkg.github.com/owner/` to use your repo owner for `owner`.
```

Comment:
`docker.pkg.github.com/owner/` seems to have gone away, but I'm not sure how this should be refactored.

Reply:
Ah, I think I get it.